src/drive/web/modules/navigation/FileExplorer.jsx

```diff
@@ -219,12 +219,11 @@ export const handleFileOpen = (file, availableOffline, props, dispatch) => {
   }
 }
 
-const appsQuery = client => client.all('io.cozy.apps')
 export default compose(
   translate(),
   queryConnect({
     apps: {
-      query: appsQuery,
+      query: Q('io.cozy.apps'),
```

Comment:
I think you have to do `() => Q('io.cozy.apps')`

Reply:
Hmm it seems to work without  🤔

Reply:
Ah oui non c'est OK https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/Query.jsx#L28-L29